### PR TITLE
Create new markdown file and file structure

### PR DESCRIPTION
**Pull request by Pezi777**

### CODE REVIEW

File: user/multi-cpu.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  include:
+   - os: linux
+     dist: bionic
+   - os: linux
+     dist: xenial
+```
+
+Would results in Bionic and Xenial jobs being run only on `amd64` architecture.
+
+The arm64 CPU architecture build job is run in an LXD compliant Linux OS image.
```
Comment (MariadeAnton) [inline user/multi-cpu.md]: Do we have the Linux OS image we'll use @michal-at-travisci ?

**Comment by vielmetti** [comment]
As a matter of style, "Packet Cloud" should be simply "Packet" throughout, e.g. `run on [Packet](https://www.packet.com)`. When we have @worksonarm new web site available we can also link and refer there.

File: user/multi-cpu-architectures.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+---
+title: Building on Multiple CPU Architectures
+layout: en
+permalink: /user/multi-cpu-architectures/
+---
+
+> This is an alpha stage of this feature and we are [eager to hear back from you](https://travis-ci.community/t/about-the-arm-cpu-architecture-category/5336). The relevant keys structure in .travis.yml may be further adapted on short notice.
+{: .alpha}
+
+> `Arm`-based building is only available for Open Source repositories (at both travis-ci.org and travis-ci.com). While available to all Open Source repositories, the concurrency available of `Arm`-based jobs is limited during the alpha period. An attempt to run `Arm`-based build for a e.g. private repository will result in a build run on standard, non-`Arm` infrastructure. For any commercial queries with regards to multi-arch builds before they are available, please contact [us](support@travis-ci.com).
```
Comment (MariadeAnton) [inline user/multi-cpu-architectures.md]:
```suggestion
> `Arm`-based building is only available for Open Source repositories (at both travis-ci.org and travis-ci.com). While available to all Open Source repositories, the concurrency available for `Arm`-based jobs is limited during the alpha period. 
> An attempt to run `Arm`-based build for a private repository will result in a build run on standard, non-`Arm` infrastructure. For any commercial queries with regards to multi-arch builds before they are available, please contact [us](support@travis-ci.com).
```

File: user/multi-cpu-architectures.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
+---
+title: Building on Multiple CPU Architectures
+layout: en
+permalink: /user/multi-cpu-architectures/
+---
+
+> This is an alpha stage of this feature and we are [eager to hear back from you](https://travis-ci.community/t/about-the-arm-cpu-architecture-category/5336). The relevant keys structure in .travis.yml may be further adapted on short notice.
```
Comment (MariadeAnton) [inline user/multi-cpu-architectures.md]:
```suggestion
> This is an alpha stage of this feature and we are [eager to hear back from you](https://travis-ci.community/t/about-the-arm-cpu-architecture-category/5336). The definition keys used in the `.travis.yml` file may be further adapted on short notice.
```

File: user/multi-cpu-architectures.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+> This is an alpha stage of this feature and we are [eager to hear back from you](https://travis-ci.community/t/about-the-arm-cpu-architecture-category/5336). The definition keys used in the `.travis.yml` file may be further adapted on short notice.
+{: .alpha}
+
+> `Arm`-based building is only available for Open Source repositories (at both travis-ci.org and travis-ci.com). While available to all Open Source repositories, the concurrency available for `Arm`-based jobs is limited during the alpha period. 
+> An attempt to run `Arm`-based build for a private repository will result in a build run on standard, non-`Arm` infrastructure. For any commercial queries with regards to multi-arch builds before they are available, please contact [us](support@travis-ci.com).
+
+If your code is used on multiple CPU architectures it probably should be tested on multiple CPU architectures. Travis CI can test on amd64 and arm64 (run on ARMv8 compliant CPUs) if the operating system is Linux.
+
+To enable testing on multiple operating systems add the `os` key to your `.travis.yml`:
```
Comment (MariadeAnton) [inline user/multi-cpu-architectures.md]:
```suggestion
To enable testing on multiple CPU architectures add the `arch` key to your `.travis.yml`:
```

File: user/multi-cpu-architectures.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+If your code is used on multiple CPU architectures it probably should be tested on multiple CPU architectures. Travis CI can test on amd64 and arm64 (run on ARMv8 compliant CPUs) if the operating system is Linux.
+
+To enable testing on multiple CPU architectures add the `arch` key to your `.travis.yml`:
+
+```yaml
+arch:
+  - amd64
+  - arm64
+os: linux
```
Comment (MariadeAnton) [inline user/multi-cpu-architectures.md]:
```suggestion
os: linux  # different CPU architectures are only supported on Linux
```

File: user/multi-cpu-architectures.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+  - arm64
+os: linux  # different CPU architectures are only supported on Linux
+```
+{: data-file=".travis.yml"}
+
+If you are already using a [build matrix](/user/customizing-the-build/#build-matrix) to test multiple versions, the `arch` key also multiplies the matrix.
+
+## Example Multi Architecture Build Matrix
+
+Here’s an example of a `.travis.yml` file using the `arch` key to compile against both `amd64` and `arm64` under Linux.
```
Comment (MariadeAnton) [inline user/multi-cpu-architectures.md]:
```suggestion
Here’s an example of a `.travis.yml` file using the `arch` key to compile against both `amd64` and `arm64` under Linux and using C as the programming language.
```

File: user/multi-cpu-architectures.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+install: skip
+
+script:
+  - cd src
+  - make all
+```
+{: data-file=".travis.yml"}
+
+Which would create 2x2 [build matrix](/user/customizing-the-build/#build-matrix): compilers x each architecture.
```
Comment (MariadeAnton) [inline user/multi-cpu-architectures.md]:
```suggestion
The `.travis.yml` file above would create 2x2 [build matrix](/user/customizing-the-build/#build-matrix): compilers x each architecture.
```

File: user/multi-cpu-architectures.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+script:
+  - cd src
+  - make all
+```
+{: data-file=".travis.yml"}
+
+The `.travis.yml` file above creates a 2x2 [build matrix](/user/customizing-the-build/#build-matrix): compilers x each architecture.
+
+There are many options available and using the `matrix.include` key is essential to include any specific entries. For example, this matrix would route builds to the arm64 and amd64 architecture queues:
```
Comment (MariadeAnton) [inline user/multi-cpu-architectures.md]:
```suggestion
There are many options available and using the `matrix.include` key is essential to include any specific entries. For example, this matrix would route builds to the arm64 and amd64 architecture environments:
```

File: user/multi-cpu-architectures.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+```yaml
+matrix:
+  include:
+   - os: linux
+     arch: amd64
+   - os: linux
+     arch: arm64
+```
+
+> Please note, that explicitly included builds inherit the first value in an array, i.e.
```
Comment (MariadeAnton) [inline user/multi-cpu-architectures.md]:
```suggestion
> Please note, that explicitly included builds inherit the first value in an array, for example:
```

**Comment by vielmetti** [comment]
Thanks everyone!  💯

**Pull request merged**